packages/code-du-travail-api/src/server/routes/search/utils.js

```diff
@@ -51,14 +51,19 @@ const mergePipe = (a, b, max_result) => {
 const getDataFromUrl = (url) => {
   const [, sourceRoute, slug] = url.split("/");
   let source = getSourceByRoute(sourceRoute);
+  // since theme routing has changed, we will keep this monkey patch for a while
+  let trimmedSlug = slug;
+  if (source === SOURCES.THEMES) {
+    trimmedSlug = slug.replace(/^\d*-/, "");
```

Comment:
it should be \d+-  not \d* since we only want to match string that match with a number followed by a "-"

Reply:
Yup it's safer you're right. It will never missmatch yet though, because no theme starts with "-" but it could happen one day